src/main/java/com/facebook/openwifirrm/modules/RRMScheduler.java

```diff
@@ -74,15 +75,22 @@ public class RRMScheduler {
 	/** The scheduler instance. */
 	private Scheduler scheduler;
 
-	/** The zones with active triggers scheduled. */
-	private Set<String> scheduledZones;
+	/**
+	 * The job keys with active triggers scheduled. Job keys take the format of
+	 * <zone>:<index>, where index applies only if there are multiple schedules for
+	 * one zone
```

Comment:
* no need for the text about `index` - it's a bit misleading anyway
* pretty sure `<zone>:<index>` will render incorrectly because `<>` are getting parsed as html tags - pick a different character or put it in a code block